doc/user/content/sql/drop-type.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "DROP TYPE"
+description: "`DROP TYPE` removes a user-defined data type."
+menu:
+  main:
+    parent: 'sql'
+---
+
+{{< version-added v0.7.1 />}}
+
+`DROP TYPE` removes a user-defined [custom data type](../create-type). It cannot be used on default data types.
+
+## Syntax
+
+{{< diagram "drop-type.svg" >}}
+
+Field | Use
+------|-----
+_data_type_name_ | The name of the type to remove.
```

Comment:
To put the underscore between italics in Markdown you need to use `&lowbar;`; you can find examples of this in most other SQL grammar docs

Reply:
Depends on the Markdown renderer, actually! Our docs site and GH. Are smart enough to handle this just fine. 🤷🏽‍♂️

Reply:
Must have gotten smarter; this didn't work when I create the docs site. Good to know!

Reply:
It worked in the test local build.

Reply:
Yeah, you and Nikhil are right here. I didn't realize the renderer had changed at some point.